react/features/invite/actions.any.js

```diff
@@ -165,9 +165,10 @@ export function updateDialInNumbers() {
         const state = getState();
         const { dialInConfCodeUrl, dialInNumbersUrl, hosts }
             = state['features/base/config'];
+        const { numbersFetched } = state['features/invite'];
         const mucURL = hosts && hosts.muc;
 
-        if (!dialInConfCodeUrl || !dialInNumbersUrl || !mucURL) {
+        if (!dialInConfCodeUrl || !dialInNumbersUrl || !mucURL || numbersFetched) {
```

Comment:
can you move the numberssFetched check to be first? Also please adjust the commenbt below.

Reply:
Optimization FTW!